### ISSUE TITLE
Shipping label error button appears before the request is made to the carrier.

### ISSUE THREAD

**Issue by AjinkyaM1** [hotwax/fulfillment#730]
## What is the motivation for adding/enhancing this feature?
We are not sending the request for the shipping label to the carrier if the facility is not in the `AUTO_SHIPPING_LABEL` group at the time of picking the order. However, when the order reaches the 'In-Progress' tab, the button for `Shipping label error` is visible on the order card. If the request has not been made, it would be helpful to ensure that the button is not displayed.

**Jam**: https://jam.dev/c/3ce3dbd5-d564-491f-9f84-e23be64ee892

**Comment by dt2patel** [comment]
@R-Sourabh what is the main reason for this being on hold?

**Comment by R-Sourabh** [comment]
@dt2patel Sorry, sir, but I don't remember the exact reason for putting this ticket on hold. Let me check and work on fixing it.

**Comment by R-Sourabh** [comment]
After debugging the issue, I found a few ways to handle this:  
- We can add a check on the facility group (`AUTO_SHIPPING_LABEL`), which will conflict when the user manually generates the shipping label and is not part of the facility group.  
- Another approach is to check the gateway message field, which we fetch in the shipping label error modal to display the error message. However, this only contains errors generated by the carrier and does not include system-generated errors.

So basically, we haven't yet concluded the discussion and still don't have a definitive way to handle this.

**Comment by swati1717** [comment]
Verified the changes in **dev** 
[Here is the Jam for reference ](https://jam.dev/c/37064689-bee1-4440-b754-780361fced5b)
